index.bs

```diff
@@ -1046,101 +1010,467 @@ BigIntValue = {
+         <code>string</code>.
+
+    <dt>|type| is the string "<code>map</code>"
+    <dd>
+      1. Let |deserialized key-value list| be a result of [=trying=] to
+         [=deserialize key-value list=] with |value|.
+
+      1. Return [=success=] with data set to a new [=Map=] object constructed with
+         |deserialized key-value list| as <code>iterable</code>.
+
+    <dt>|type| is the string "<code>object</code>"
+    <dd>
+      1. Let |deserialized key-value list| be a result of [=trying=] to
+         [=deserialize key-value list=] with |value|.
+
+      1. Return [=success=] with data set to a result of [=Object.fromEntries=] with
+         |deserialized key-value list| as <code>iterable</code>.
+
+    <dt>|type| is the string "<code>regexp</code>"
+    <dd>
+      1. Let |pattern| be the value of the <code>pattern</code> field of |local protocol value|.
+
+      1. Let |flags| be the value of the <code>flags</code> field of |local protocol value| or
+         undefined if no such a field.
+
+      1. Return [=success=] with data set to a new [=RegExp=] object constructed with |pattern| and
+         |flags|.
+
+    <dt>|type| is the string "<code>set</code>"
+    <dd>
+      1. Let |deserialized value list| be a result of [=trying=] to
+         [=deserialize value list=] with |value|.
+
+      1. Return [=success=] with data set to a new [=Set object=] constructed with
+         |deserialized value list| as <code>[iterable]</code>.
+
+    <dt>otherwise
+    <dd>Return [=error=] with [=error code=] [=invalid argument=].
+  </dl>
+
+</div>
+
+### Remote Value ### {#data-types-protocolValue-RemoteValue}
+
+Values accessible from the ECMAScript runtime are represented by a mirror
+object, specified as <code>RemoteValue</code>. The value's type is specified in
+the <code>type</code> property. In the case of JSON-representable primitive
+values, this contains the value in the <code>value</code> property; in the case
+of non-JSON-representable primitives, the <code>value</code> property contains a
+string representation of the value. For non-primitive objects, the
+<code>objectId</code> property contains a string id that provides a unique
+handle to the object, valid for its lifetime inside the engine. For some
+non-primitive types, the <code>value</code> property contains a representation
+of the data in the ECMAScript object; for container types this can contain
+further <code>RemoteValue</code> instances. The <code>value</code> property can
+be null if there is a duplicate object i.e. the object has already been
+serialized in the current <code>RemoteValue</code>, perhaps as part of a
+cycle, or otherwise when the maximum serialization depth is reached.
+
+[=Nodes=] are also represented by <code>RemoteValue</code> instances. These have
+a partial serialization of the node in the value property.
+
+Note: mirror objects do not keep the original object alive in the runtime. If an
```

Comment:
I know you just moved this but I don't think we've actually agreed that's true.

Reply:
Added an issue to reconsider the lifecycle.